ports/mimxrt/machine_pin.c

```diff
@@ -280,6 +281,22 @@ STATIC void machine_pin_obj_print(const mp_print_t *print, mp_obj_t o, mp_print_
     mp_printf(print, "Pin(%s)", qstr_str(self->name));
 }
 
+STATIC mp_uint_t machine_pin_ioctl(mp_obj_t self_in, mp_uint_t request, uintptr_t arg, int *errcode) {
```

Comment:
please shift this function down to just before the definition of `machine_pin_obj_protocol` (like how it is in the rp2 port)

Reply:
fixed